src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java

```diff
@@ -385,15 +386,26 @@ public void validateUserCanRead(
     for (String bucket : buckets) {
       List<Boolean> permissions = List.of();
       try {
+        logger.info(
+            "Checking {} permission on bucket {} with options for user project {}",
+            GCS_SOURCE_BUCKET_REQUIRED_PERMISSION,
+            bucket,
+            cloudEncapsulationId);
```

Comment:
Do you understand how the user project (A requester pays concept) is impacted by the pet account/source bucket?